src/main/resources/application.yaml

```diff
@@ -12,3 +12,8 @@ management:
 spring:
   application:
     name: Blob Router Service
+
+accesstoken:
+  serviceConfig:
+    - serviceName: bulkscan
```

Comment:
came case  or kebab-case?

Reply:
those will be all lowercase - same as we have currently in processor. please check bulk-scan-shared-infra for references ;)

Reply:
oh sorry, i think i confused myself. clarified with @aliveni's help